resources/victron-virtual-functions.js

```diff
@@ -212,6 +262,19 @@
       '<div><strong>Most relevant path(s):</strong><ul><li><code>/SwitchableOutput/output_1/State</code> &mdash; Requested on/off state of channel, separate from dimming.</li><li><code>/SwitchableOutput/output_1/Alarm</code> &mdash; Indicates if an alarm condition is present (e.g., high water level).</li><li><code>/SwitchableOutput/output_1/Measurement</code> &mdash; If supported by the connected device, this path may provide additional measurement data, such as water level percentage.</li></ul></div>',
       '<div><strong>Outputs:</strong><ol><li><code>Passthrough</code> &mdash; Outputs the original <tt>msg.payload</tt> without modification</li><li><code>State</code> &mdash; <tt>msg.payload</tt> contains a <tt>0</tt> or <tt>1</tt> representing the on/off state of the pump</li></ol></div>',
       '/resources/@victronenergy/node-red-contrib-victron/docs/bilge_pump.png'
+    ),
+    [victronVirtualConstantsExports.SWITCH_TYPE_MAP.RGB_COLOR_WHEEL]: createDocTemplate(
+      `<div><strong>Most relevant path(s):</strong><ul>
+      <li><code>/SwitchableOutput/output_1/State</code> &mdash; Requested on/off state of the light.</li>
+      <li><code>/SwitchableOutput/output_1/LightControls</code> &mdash; Array of 5 doubles: <tt>[Hue (0-360°), Saturation (0-100%), Brightness (0-100%), White (0-100%), ColorTemperature (0-6500K)]</tt>.
+        <br><span style="font-size:0.95em;color:#666;">Array elements used depend on selected control types:<br>
+        • RGB color wheel: Hue, Saturation, Brightness<br>
+        • CCT wheel: Brightness, ColorTemperature<br>
+        • RGB + white dimmer: Hue, Saturation, Brightness, White</span>
+      </li>
```

Comment:
As this is not a trivial value to set, maybe add an example object with a valid payload, something like:

```javascript
msg = {
    payload: {
        '/SwitchableOutput/output_1/LightControls': [150.5, 50, 50, 0, 0]
    }
}
```